core/src/test/scala/tech/beshu/ror/unit/acl/factory/LocalUsersTest.scala

```diff
@@ -140,6 +140,55 @@ class LocalUsersTest extends AnyWordSpec with Inside {
           allUsersResolved(Set(User.Id("admin"), User.Id("dev")))
         )
       }
+      "users section defined without wildcard patterns" in {
+        val config =
+          s"""
+             |readonlyrest:
+             |  access_control_rules:
+             |  - name: test_block1
+             |    auth_key: admin:container
+             |
+             |  users:
+             |  - username: user1
+             |    groups: ["group1", "group3"]
+             |    auth_key: "user1:pass"
+             |
+             |  - username: user2
+             |    groups: ["group2", "group4"]
+             |    auth_key: "user2:pass"
+             |
+             |  - username: user4
+             |    groups: ["group5", "group6"]
+             |    auth_key: "user4:pass"
+             |""".stripMargin
+
+        assertLocalUsersFromConfig(config, allUsersResolved(Set(
+          User.Id("user1"), User.Id("user2"), User.Id("user4"), User.Id("admin")
+        )))
+      }
+      "impersonators section defined with users" in {
```

Comment:
do we have a `groups` rule tested for local users? (AFAIR, Simone complained about this case)